pom.xml

```diff
@@ -2557,7 +2576,7 @@
         <commons-lang.wso2.version>2.6.0.wso2v1</commons-lang.wso2.version>
 
         <!--  Test dependencies -->
-        <carbon.automation.version>4.4.11</carbon.automation.version>
+        <carbon.automation.version>4.4.12-SNAPSHOT</carbon.automation.version>
```

Comment:
Update the version once https://github.com/wso2/carbon-platform-integration/pull/102 is merged and released.